tests/test_welcome_plugin.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from unittest.mock import AsyncMock, patch
+from src.lettuce.plugins.welcome.welcome import WelcomePlugin  # Adjust the import according to your project structure
```

Comment:
This should work just fine:
```suggestion
from lettuce.plugins.welcome.welcome import WelcomePlugin
```